Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
in case anyone else was curious about where this `version` came from: https://internals.rust-lang.org/t/upcoming-changes-to-cargo-lock/14017

Reply:
You have a keen eye for details.